diff --git a/node_modules/phaser3-nineslice/src/NineSlice.js b/node_modules/phaser3-nineslice/src/NineSlice.js
index 56d34b1..31f73e1 100644
--- a/node_modules/phaser3-nineslice/src/NineSlice.js
+++ b/node_modules/phaser3-nineslice/src/NineSlice.js
@@ -1,4 +1,7 @@
-import Phaser from 'phaser'
+import 'phaser/src/gameobjects/image/ImageCreator'
+import RenderTexture from 'phaser/src/gameobjects/rendertexture/RenderTexture'
+import EventEmitter from 'phaser/src/events/EventEmitter'
+import Rectangle from 'phaser/src/geom/rectangle/Rectangle'
 
 import murmur from './murmur'
 
@@ -65,7 +68,7 @@ export const EVENTS = {
   UPDATE_SAFE_BOUNDS: 'updatesafebounds',
 }
 
-export default class NineSlice extends Phaser.GameObjects.RenderTexture {
+export default class NineSlice extends RenderTexture {
   /**
    * @param {Phaser.Scene} scene the parent scene for this NineSlice
    * @param {SliceConfig} _sliceConfig specifies details of where we source
@@ -84,9 +87,9 @@ export default class NineSlice extends Phaser.GameObjects.RenderTexture {
     this.updateSafeBounds = this.updateSafeBounds.bind(this)
     this.enableDebugDraw = this.enableDebugDraw.bind(this)
 
-    this.events = new Phaser.Events.EventEmitter()
+    this.events = new EventEmitter()
     this.sliceConfig = defaultSliceConfig(_sliceConfig)
-    this._safeBounds = new Phaser.Geom.Rectangle()
+    this._safeBounds = new Rectangle()
 
     const { sourceKey, sourceFrame } = this.sliceConfig
     this.sourceTex = scene.sys.textures.get(this.sliceConfig.sourceKey)
diff --git a/node_modules/phaser3-nineslice/src/Plugin.js b/node_modules/phaser3-nineslice/src/Plugin.js
index 6ca398e..7552fd1 100644
--- a/node_modules/phaser3-nineslice/src/Plugin.js
+++ b/node_modules/phaser3-nineslice/src/Plugin.js
@@ -1,8 +1,8 @@
-import Phaser from 'phaser'
+import BasePlugin from 'phaser/src/plugins/BasePlugin'
 import NineSlice from './NineSlice'
 import processOffsetsArray from './processOffsetsArray'
 
-export default class Plugin extends Phaser.Plugins.BasePlugin {
+export default class Plugin extends BasePlugin {
   constructor(mgr) {
     super(mgr)
     mgr.registerGameObject('nineslice', this.addNineSlice, this.makeNineSlice)
